summary_workflows/identification/specification/I_benchmark_specification.md

```diff
@@ -63,60 +72,71 @@ Fields:
+The plot should be divided into diagonal quartiles based on the distance from optimal performance.
+Alternatively, if the plot is divided into square quartiles, the following ranking order should be applied: top-right, bottom-right, top-left, bottom-left.
 
-Ranking: The best performing tool is the one with the highest precision value.
-
-2. **2D scatter plot** visualizing **TPR and FPR** of poly(A) site identification. Separate plots should be prepared for different values of distance threshold:
-
-- 10 nt
-- 50 nt
-- 100 nt
+2. **bar plot** visualizing **AUC** of Precision-Recall curve with single AUC value for each tool.
 
-**X axis** - FPR(d)  
-**Y axis** - TPR(d)  
-where _d - distance threshold_
+**X axis** - benchmarked tool  
+**Y axis** - AUC
 
-Input datasets:
+Input data:
 
 - RNA-Seq data compared with 3'end sequencing data
 - Simulated RNA-Seq data compared with dataset used for simulation
 
-Ranking: The best performing tool is the one with the highest TPR combined with lowest FPR (top left part of the plot) and the worst performing tool is the one with the lowest TPR combined with highest FPR (bottom left part of the plot. The plot should be divided into diagonal quartiles based on the distance from optimal performance. Alternatively, if the plot is divided into square quartiles, the following ranking order should be applied: top-left, top-right, bottom-left, bottom-right.
+Ranking: The best performing tool is the one with the highest AUC value.
 
-3. **bar plot** visualizing **AUC** of ROC plot with single AUC value for each tool.
+3. **bar plot** visualizing **proportion of PAS assigned to 3'-UTRs**.
 
 **X axis** - benchmarked tool  
-**Y axis** - AUC
+**Y axis** - proportion of PAS assigned to 3'-UTRs
 
 Input data:
 
 - RNA-Seq data compared with 3'end sequencing data
 
+Ranking: The best performing tool is the one with the highest proportion of PAS assigned to 3'-UTRs
 
-Ranking: The best performing tool is the one with the highest AUC value.
+4. **bar plot** visualizing **percentage of genes with correctly identified number of PAS**
+
+**X axis** - benchmarked tool  
+**Y axis** - percentage of genes with correctly identified number of PAS
+
+Input data:
+
+- RNA-Seq data compared with 3'end sequencing data
+
+Ranking: The best performing tool is the one with the highest ercentage of genes with correctly identified number of PAS.
 
-Optional plots:
+Optional plots - can be generated outside of OpenEBench:
 
-4. **2D line plot** visualising **TPR and FPR** in the form of [ROC curve][roc-curve], i.e. TPR and FPR calculated for a range of distance thresholds are plotted together. 
+1. **2D line plot** visualising **TPR and FPR** in the form of [ROC curve][roc-curve], i.e. TPR and FPR calculated for a range of distance thresholds are plotted together. 
 
 **X axis** - FPR(d)  
```

Comment:
I don't think we can calculate FPR=FP / (FP + TN), as it contains true negatives which we cannot assess. Please correct me if I'm wrong!

Reply:
See above

Reply:
Before this version of specifications I realised that TN doesn't really work and removed it from all "obligatory" metrics, but I forgot to update the optional plots. This should be Precision-Recall curve instead of ROC.